examples/palace-info.py

```diff
@@ -37,6 +37,19 @@
 
     print(f'\nInfo of device {args.device.name!r}:')
     print('ALC version: {}.{}'.format(*args.device.alc_version))
+
+    # TODO: AL info
+    with Device() as dev, Context(dev) as ctx:
```

Comment:
No need for another device here.

Reply:
Oh wait, `args.device` is already a device, didn't notice that.